src/renderers/WebGLRenderer.js

```diff
@@ -1381,11 +1381,15 @@ function WebGLRenderer( parameters ) {
 
 			}
 
-			var children = object.children;
+			if ( ! object.layers.recursive || object.layers.recursive && layerTest ) {
```

Comment:
I guess ideally this would simply be:

```js
if ( object.layers.recursive )
```

Is this why you were proposing `recursive = true` by default?

Reply:
I think I'm open to make it `true` by default.

Reply:
This condition is the way it is to accommodate the two cases of "traverse and render all children regardless of parent layer" (`layers.recursive = false`) and "only traverse and render children if parent layer passes test" (`layers.recursive = true`).

I originally suggested _only_ supporting the `layers.recursive = true` behavior which would change this condition to be:

```js
if ( layerTest ) {
```